packages/core/src/components/MenuToggle/index.tsx

```diff
@@ -161,29 +164,36 @@ export class MenuToggle extends React.Component<Props & WithStylesProps, State>
 
     const Button = muted ? MutedButton : BaseButton;
 
+    const configuredToggleButton = toggleIcon ? (
```

Comment:
maybe `toggleButton` to simplify things?

Reply:
this could be a `let toggleButton` with the default value being the ternary. then a
```
if (toggleTrigger) {
  toggleButton = <span onClick={this.handleToggleMenu}>{toggleTrigger}</span>'
}
```

instead of lines 188 - 192

Reply:
also, can you make this a button with styles reset? having onClicks on elements like spans and buttons isn't great for accessibility

Reply:
nice yeah!